README.md

```diff
@@ -117,326 +68,113 @@ https://pmi.pay1.de/
    automated testing, demo systems etc.)
  * Set the hashing algorithm to "md5 oder sha2-384 (für Migration)" ("advanced" tab in the portal config)
  * Put the notification listener URL of where you will deploy the microservice into "Transaction Status URL" in the 
-   "advanced" tab of the portal. The value typically is https://{your-service-instance.example.com}/payone/notification .  
+   "advanced" tab of the portal. The value typically is https://{your-service-instance.example.com}/{tenant-name}/payone/notification .  
  * Configure the "riskcheck" settings as intended (esp. 3Dsecure)
 
 > Do not use a merchant account across commercetools projects, you may end up mixing customer accounts (debitorenkonten). 
 
 ### Configuration of the Integration Service itself
 
-The integration service requires - _unless otherwise stated_ - the following environment variables. 
+The integration service requires - _unless otherwise stated_ - the following environment variables 
+or Java runtime arguments. 
+**Note**, Java runtime arguments, e.g. those supplied with `-D` prefix, have precedence over environment variables 
+with the same name. 
 
-At the end of this README you can find a copy/paste shell template that sets the variables. Alternatively you can use
-[a properties file](#appendix-2-alternative-configuration-via-properties-file).
+### Mandatory common properties
```

Comment:
depends on how you supply them: it might be Java property or environment variable. Common name is "properties"

See https://github.com/commercetools/commercetools-payone-integration/blob/b920354730aa9271797a3721f69c7f9579b47c79/service/src/main/java/com/commercetools/pspadapter/payone/config/PropertyProvider.java#L66-L66